src/renderers/dom/shared/ReactBrowserEventEmitter.js

```diff
@@ -132,40 +139,44 @@ var ReactBrowserEventEmitter = Object.assign({}, ReactEventEmitterMixin, {
    * @param {string} registrationName Name of listener (e.g. `onClick`).
    * @param {object} contentDocumentHandle Document which owns the container
    */
-  listenTo: function(registrationName, contentDocumentHandle) {
+  listenTo: function(registrationName, contentDocumentHandle, node) {
     var mountAt = contentDocumentHandle;
-    var isListening = getListeningForDocument(mountAt);
+    var isDocListening = getListeningForDocument(mountAt);
     var dependencies =
       EventPluginRegistry.registrationNameDependencies[registrationName];
 
     for (var i = 0; i < dependencies.length; i++) {
       var dependency = dependencies[i];
-      if (
-        !(isListening.hasOwnProperty(dependency) && isListening[dependency])
+
+      if (localOnly.hasOwnProperty(dependency)) {
+        ReactDOMEventListener.trapBubbledEvent(
```

Comment:
Can this be called more than once for a single node? It seems like it would since ensureListeningTo is called on updates. Let me know if I'm missing something.